build-logic/src/main/kotlin/polaris-java.gradle.kts

```
@@ -153,18 +155,22 @@ dependencies {
       GradleException("assertj-core not declared in libs.versions.toml")
     }
   )
-  testFixturesImplementation(
+  val mockitoCoreLib =
     libs.findLibrary("mockito-core").orElseThrow {
       GradleException("mockito-core not declared in libs.versions.toml")
     }
-  )
+
+  testFixturesImplementation(mockitoCoreLib)
+
+  mockitoAgent(mockitoCoreLib) { isTransitive = false }
 }
 
 tasks.withType<Test>().configureEach {
   systemProperty("file.encoding", "UTF-8")
   systemProperty("user.language", "en")
   systemProperty("user.country", "US")
   systemProperty("user.variant", "")
+  jvmArgs("-javaagent:${mockitoAgent.asPath}")
```

Should use this to let the system specific path not pollute the build cache.
```suggestion
    jvmArgumentProviders.add( CommandLineArgumentProvider {  listOf("-javaagent:${mockitoAgent.asPath}") })
```